### ISSUE TITLE
How to train on a custom dataset 

### ISSUE THREAD

**Issue by MyVanitar** [YihangLou/fast-rcnn-train-another-dataset#5]
Hello,

How can we pick-up a pre-trained model and re-train it on a custom data-set (user defined data-set)? I have selected the PVANET+ (Compressed) because as I see it shows a good combination of speed and mAP.

Questions like:

   

- The Train/Test images should be annoated to what format?
- How to created the LMDB files for PVANet after annotation? By standard Caffe tools or you have your own tool?
- How to modify CFG / Prototxt files? 

Considering that I gonna fine-tune on existing trained `.caffemodels`, so I must get a high accuracy even on a small dataset. Yes?
